dashboard/test/test_helper.rb

```diff
@@ -79,6 +79,7 @@ class ActiveSupport::TestCase
 
   teardown do
     Dashboard::Application.config.action_controller.perform_caching = false
+    I18n.locale = I18n.default_locale
```

Comment:
Does it make more sense to reset in tear down, or in setup? (not sure of answer)